libs/core/src/lib/list/list-item/list-item.component.ts

```diff
@@ -0,0 +1,103 @@
+import {
+    ChangeDetectionStrategy,
+    ChangeDetectorRef,
+    Component,
+    ContentChild,
+    ElementRef,
+    EventEmitter,
+    HostBinding,
+    HostListener,
+    Input,
+    Output,
+    ViewEncapsulation
+} from '@angular/core';
+import { CheckboxComponent } from '../../checkbox/checkbox/checkbox.component';
+import { DefaultMenuItem } from '../../menu/default-menu-item.class';
+import { RadioButtonComponent } from '../../radio/radio-button/radio-button.component';
+
+/**
+ * The component that represents a list item.
+ * The list item can contain plain text, links or actions.
+ */
+@Component({
+    // tslint:disable-next-line:component-selector
+    selector: '[fdListItem] ,[fd-list-item]',
+    templateUrl: './list-item.component.html',
+    host: {
+        class: 'fd-list__item',
+        role: 'listitem'
+    },
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    encapsulation: ViewEncapsulation.None
+})
+export class ListItemComponent implements DefaultMenuItem {
+    /** Whether list item is selected */
+    @Input()
+    @HostBinding('attr.aria-selected')
+    @HostBinding('class.is-selected')
+    selected = false;
+
+    /** Whether list item contains link */
+    @Input()
+    @HostBinding('class.fd-list__item--link')
```

Comment:
Similar to another comment I don't think this input is necessary, just check if there is a fd-list-link contentchild

Reply:
It will be beneficial and it will make list easier for users. The only thing that bothers me is what happen, when someone has 
anchor with ListLinkDirective, which has some `*ngIf`. The changes of markup won't be noticed, because it's almost impossible to track it on ContentChild

Reply:
Let's have a talk about it on meeting